server/routes/updateCommunity.ts

```diff
@@ -18,15 +18,16 @@ const updateCommunity = async (models, req: UserRequest, res: Response, next: Ne
         offchain_community_id: community.id,
       },
     });
-    console.log(userMembership);
-    if (userMembership.role.permission !== 'admin') {
-      return next(new Error('Invalid community or chain'));
+    if (userMembership.permission !== 'admin') {
+      return next(new Error('Invalid Permission to Update Community'));
     }
   }
 
-  if (req.body.name) community.setName(req.body.name);
-  if (req.body.title) community.setDescription(req.body.description);
+  if (req.body.name) community.name = req.body.name;
+  if (req.body.description) community.description = req.body.description;
   if (req.body['featured_tags[]']) community.featured_tags = req.body['featured_tags[]'];
+  if (req.body.invites) community.invitesEnabled = !community.invitesEnabled;
+  if (req.body.privacy) community.privacyEnabled = !community.privacyEnabled;
```

Comment:
Are we toggling invitesEnabled and privacyEnabled based on what fields come from the client? That seems confusing and likely to produce issues down the line.

Reply:
It's toggling only if the value from the client is `true`, otherwise, it stays the same. It's to get around the issue of passing false or null from the client to server, it comes as a string not a boolean.

Reply:
This came up elsewhere today with @drewstone. I think we'll want to update this code per the latest changes that have landed on master.